gtdynamics/universal_robot/sdf.cpp

```diff
@@ -75,19 +75,18 @@ JointParams ParametersFromSdfJoint(const sdf::Joint &sdf_joint) {
 Pose3 GetJointFrame(const sdf::Joint &sdf_joint,
                     const LinkSharedPtr &parent_link,
                     const LinkSharedPtr &child_link) {
-  auto frame = sdf_joint.PoseFrame();
-  if (frame == "" || frame == child_link->name()) {
-    if (sdf_joint.Pose() == ignition::math::Pose3d())
+  auto frame = sdf_joint.PoseRelativeTo();
+  Pose3 sdf_joint_pose = Pose3FromIgnition(sdf_joint.RawPose());
+  if (frame.empty() || frame == child_link->name()) {
+    if (sdf_joint.RawPose() == ignition::math::Pose3d()) {
       return child_link->wTl();
-    else
-      return child_link->wTl() * Pose3FromIgnition(sdf_joint.Pose());
+    } else {
+      return child_link->wTl() * Pose3FromIgnition(sdf_joint.RawPose());
```

Comment:
Can you
(a) document what this case means?
(b) assign `Pose3FromIgnition(sdf_joint.RawPose())` to the standard `T` conventions so I can check the indices match?